pymemcache/client/base.py

```diff
@@ -739,6 +744,22 @@ def _raise_errors(self, line, name):
             error = line[line.find(b' ') + 1:]
             raise MemcacheServerError(error)
 
+    def _key_helper(self, key, remapped_keys, prefixed_keys):
+        """
+        This function is extracted from _fetch_cmd to support
+            future extension for remcached.
+        Remcached will support rich data structure like list/map/set,
```

Comment:
This class need not be aware of potential subclass implementations. Same as above, let's remove the reference here.